xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -1052,6 +1052,9 @@ bool CDVDVideoCodecFFmpeg::GetPictureCommon(VideoPicture* pVideoPicture)
   else
     pVideoPicture->color_range = m_hints.colorRange == AVCOL_RANGE_JPEG ? 1 : 0;
 
+  pVideoPicture->hdrType = m_hints.hdrType;
+  pVideoPicture->isHdr = (m_hints.hdrType != StreamHdrType::HDR_TYPE_NONE);
```

Comment:
for logic that simple and with values changing at the same time as new hdrType member, is it worth an extra member variable instead of an iniline member function like IsHdr()?